nextflow.config

```diff
@@ -18,6 +18,7 @@ params {
     checkm_metadata             = null
     gtdbtk_metadata             = null
     se_reads                    = false
+    rename_contigs              = false
```

Comment:
Didn't we decide to rename whenever there are duplicate names?

Reply:
that's true. I can work on that. we have the module that checks duplicates. I could link these two modules together.

Reply:
I got stuck all day trying to connect CHECK_DUPLICATES.nf with RENAME_CONTIGS.nf but I didn't succeed.
I was trying to have logic conditions where if the channel from CHECK_DUPLICATES is not empty, then just run RENAME_CONTIGS.nf but I don't find a good solution.